docs/integrations/third-party-libraries.md

```diff
@@ -37,6 +35,7 @@ This can be done using third-party libraries created by the community.
 - [vue-zustand](https://github.com/wobsoriano/vue-zustand) — State management solution for Vue based on Zustand.
 - [zoov](https://github.com/InfiniteXyy/zoov) — State management solution based on Zustand with Module-like API.
 - [zundo](https://github.com/charkour/zundo) — 🍜 Undo and redo middleware for Zustand, enabling time-travel in your apps.
+- [zustand-interval-persist](https://www.npmjs.com/package/zustand-interval-persist) — An enhancement for zustand that enables automatic saving of the store's state to the specified storage at regular interval.
```

Comment:
Oh, it looks like zundo and zukeeper should be in the other order.
zustand-interval-persist should be between zustand-forms and zustand-middleware-computed-state.

Reply:
rearranged order for zukeeper and zundo , and placed zustand-interval-persist at correct place.